modules/core/packet-server/client/cli/query.go

```diff
@@ -28,11 +28,11 @@ func getCmdQueryChannel() *cobra.Command {
 			}
 			clientID := args[0]
 
-			queryChannel := types.NewQueryClient(clientCtx)
+			queryClient := types.NewQueryClient(clientCtx)
```

Comment:
this was just a mistake in find and replace i made in the last pr that im correcting now